src/client/index.html

```diff
@@ -397,7 +399,76 @@
         });
       });
     </script>
-    <script src="lang.js"></script>
+    <script>
+      document.addEventListener("DOMContentLoaded", async function () {
+        const defaultLang = navigator.language.startsWith("ja")
```

Comment:
I think you can use Intl.Locale instead of adding new conditions for every new language code:

```suggestion
        const locale = new Intl.Locale(navigator.language);
        const defaultLang = locale.language;
```